projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -165,12 +165,15 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     public filteringOptions: IComboFilteringOptions = {
         caseSensitive: false
     };
+    /* @hidden @internal */
```

Comment:
```suggestion
    /** @hidden @internal */
```
To actually get the tags working :)
Also not a huge fan of this being underscore prefixed, kind of inconsistent with the rest